publish.ps1

```diff
@@ -68,10 +68,12 @@ function do-package()
     Copy-Item -Force "$distDir\readme.txt"            $tempDir -ErrorAction Stop
 
     # Create archive
-    7za a -tzip -mx9 $outZip "$tempDir\*"
+    7z a -tzip -mx9 $outZip "$tempDir\*"
```

Comment:
@pizza2004 7za is an application shipped with 7zip for command line operations. We are creating zip files not 7z files. We are using 7zip because its fast and efficient.